hudi-client/src/test/java/org/apache/hudi/testutils/HoodieClientTestHarness.java

```diff
@@ -66,17 +65,12 @@
   protected transient HoodieTestDataGenerator dataGen = null;
   protected transient ExecutorService executorService;
   protected transient HoodieTableMetaClient metaClient;
-  private static AtomicInteger instantGen = new AtomicInteger(1);
   protected transient HoodieWriteClient writeClient;
   protected transient HoodieReadClient readClient;
   protected transient HoodieTableFileSystemView tableView;
 
   protected final SparkTaskContextSupplier supplier = new SparkTaskContextSupplier();
 
-  public String getNextInstant() {
-    return String.format("%09d", instantGen.getAndIncrement());
-  }
-
```

Comment:
Remove this implementation due to lack of reusability; use `HoodieTestTable#makeNewCommitTime(int sequence)` instead